hist/hist/inc/THn.h

```diff
@@ -176,9 +178,9 @@ class THn: public THnBase {
 
 protected:
    TNDArrayT<Double_t> fSumw2; // bin error, lazy allocation happens in TNDArrayT
-   mutable Int_t* fCoordBuf; //! Temporary buffer
+   mutable std::vector<Int_t> fCoordBuf; //! Temporary buffer
 
-   ClassDef(THn, 1); //Base class for multi-dimensional histogram
+   ClassDef(THn, 2); // Base class for multi-dimensional histogram
```

Comment:
This no need/requirement to increase the class version if a transient member change (the version is an id for the the format of the on-file representation)

Reply:
true -- are you requesting that the version number stays the same or is it anyway good practice to bump it?

Reply:
Indeed "when in doubt, increment" (as unnecessary increment are better than missing an increment).  However increment are not completely free either (passed 16, it increase the size of the array of `StreamerInfo` pointer; a file containing both version would need to record both version (slight file size increase, slight processing time increase at file opening),    

I would lean toward keeping it at `1` but only slightly leaning.

Reply:
Does the ClassDef for THn anyways need to be incremented because of the change in TNDArrayT?

Reply:
No.  You do not need to change the Container class ClassDef if a containee change (however you currently, unfortunately, need to update the ClassDef of a derived class if the base class changes).

Reply:
Ok I've updated this with the minimal ClassDef version increments (only needed for TNDArray and TNDArrayT in the end)